### ISSUE TITLE
Intercom may return 'None' for custom attributes

### ISSUE THREAD

**Issue by teeberg** [jkeyes/python-intercom#92]
First of all, the new version 2.0 looks already really awesome and easy to use!

There is only one minor problem that I'm running into, which is:

````python
c = Company(id=network.id)
c.custom_attributes.update(enabled_features='')
c.save()
```
Intercom will respond to this query with the new network data, but it has set `enabled_features` to `None`, which raises this exception:
```
Traceback (most recent call last):
  File "test.py", line 36, in <module>
    c.save()
  File "intercom/api_operations/save.py", line 51, in save
    return self.from_response(response)
  File "intercom/traits/api_resource.py", line 60, in from_response
    self.from_dict(response)
  File "intercom/traits/api_resource.py", line 67, in from_dict
    setattr(self, attribute, value)
  File "intercom/traits/api_resource.py", line 94, in __setattr__
    value_to_set = FlatStore(value)
  File "intercom/lib/flat_store.py", line 10, in __init__
    self.update(*args, **kwargs)
  File "intercom/lib/flat_store.py", line 30, in update
    self[key] = other[key]
  File "intercom/lib/flat_store.py", line 18, in __setitem__
    "custom data only allows string and real number values")
ValueError: custom data only allows string and real number values
```

**Comment by jkeyes** [comment]
@teeberg Thank's for the nice words. A few people have reported the `custom_attributes` and `None` value issue. It's sitting in a PR waiting for me to merge.

**Comment by teeberg** [comment]
Oh awesome, thank you for the fast response! I only checked the issues which are in an exceptionally clean state. :-) I totally missed to check the PRs too. I'll just use the fork in the meantime then!

**Comment by jkeyes** [comment]
Fixed in 2.0 which I released to PyPi today.

**Comment by teeberg** [comment]
Wow that's awesome! Thanks so much for the fast work! Keep it up!
